File: app/src/main/java/com/kiquenet/introduceme/ui/di/scope/ApplicationContext.java
package com.kiquenet.introduceme.ui.di.scope;

import javax.inject.Qualifier;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

/**
 * @author n.diazgranados
 */
@Qualifier
@Retention(RetentionPolicy.RUNTIME)
public @interface ApplicationContext {
}

File: app/src/main/java/com/kiquenet/introduceme/di/scope/ApplicationContext.kt
package com.kiquenet.introduceme.di.scope

import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy
import javax.inject.Qualifier

/**
 * @author n.diazgranados
 */
@Qualifier
@Retention(RetentionPolicy.RUNTIME)
annotation class ApplicationContext